src/test/regress/expected/failure_mx_metadata_sync.out

```diff
@@ -164,6 +164,130 @@ WHERE logicalrelid='t2'::regclass;
  f
 (1 row)
 
+-- Failure to set groupid in the worker
+SELECT citus.mitmproxy('conn.onQuery(query="^UPDATE pg_dist_local_group SET groupid").cancel(' || :pid || ')');
+ mitmproxy
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT stop_metadata_sync_to_node('localhost', :worker_2_proxy_port);
+NOTICE:  dropping metadata on the node (localhost,9060)
+ERROR:  canceling statement due to user request
+SELECT citus.mitmproxy('conn.onQuery(query="^UPDATE pg_dist_local_group SET groupid").kill()');
+ mitmproxy
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT stop_metadata_sync_to_node('localhost', :worker_2_proxy_port);
+NOTICE:  dropping metadata on the node (localhost,9060)
+WARNING:  server closed the connection unexpectedly
+	This probably means the server terminated abnormally
+	before or while processing the request.
+CONTEXT:  while executing command on localhost:xxxxx
+WARNING:  connection not open
+CONTEXT:  while executing command on localhost:xxxxx
+ stop_metadata_sync_to_node
```

Comment:
I think one useful thing would be to show that pg_dist_node on the coordinator set the metadata to false (something like below, and only a single test should be fine):
```
SELECT hasmetadata FROM pg_dist_node WHERE nodeport = :worker_2_proxy_port and nodename='localhost'
```